src/checkDigits.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const checkDigits = (input, answer) => {
+  const digits = input.split('');
+  let bulls = 0;
+  let cows = 0;
+
+  if (digits.some(digit => isNaN(digit))) {
+    return 'Only numbers are allowed!';
+  }
+
+  if (digits.length !== 4) {
+    return 'You must enter 4 digits';
+  }
+
+  if ([...new Set(digits)].length !== 4) {
```

Comment:
Set has a getter `size` and you can do like this
```suggestion
  if (new Set(digits).size !== 4) {
```